pages/k8s/1.25/upgrading.md

```diff
@@ -0,0 +1,40 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Upgrading"
+  description: How to upgrade your version of Charmed Kubernetes.
+keywords: juju, upgrading, track, version
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: 1.25/upgrading.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
```

Comment:
I will add the standard upgrade instructions here, unless there are any specific things to call out